python/ray/serve/tests/test_pipeline_driver.py

```diff
@@ -100,5 +101,25 @@ def test_dag_driver_custom_schema(serve_instance):
     assert resp.json() == 100
 
 
+@serve.deployment
+def combine(*args):
+    return list(args)
+
+
+def test_dag_driver_partial_input(serve_instance):
+    with InputNode() as inp:
+        dag = DAGDriver.bind(
+            combine.bind(echo.bind(inp[0]), echo.bind(inp[1])),
+            input_schema=json_request,
+        )
+    handle = serve.run(dag)
+    assert ray.get(handle.predict.remote([1, 2])) == [1, 2]
```

Comment:
can you try it with an example that passes multiple objects to `dag.execute(1, 2, [3,4])` ? I remember default DAGDriver will throw due to mismatched signature that lead to different behaviors between ray dag and serve dag regarding InputNode handling